print-dhcp6.c

```diff
@@ -784,6 +790,43 @@ dhcp6opt_print(netdissect_options *ndo,
 			ND_PRINT(")");
 			break;
 
+		case DH6OPT_BOOTFILE_URL:
+			if (optlen < 2) {
```

Comment:
What constraint does this check implement? RFC 5970 Section 3.1 does not impose any requirements on `option-len` value. It does require `boot-file-url` to meet certain syntax requirements, is this related?

Reply:
No, there is nothing special about 2 - a 4-octet url is just as likely to be too short as a 2-octet one, so I'll just remove the check and leave the validity understanding to the user.

Reply:
That's what I thought, thank you.